SEED-platform/seed
# refactor: update bsyncr pipeline

#### Any background context you want to provide?
The bsyncr service has been updated to
1. allow configuration of the regression model type
2. return the bsync xml file as well as a plot of the fitted model in a zip

#### What's this PR do?
Modifies the bsyncr pipeline to handle these changes

#### How should this be manually tested?
Run the new migration

Pull the latest bsyncr-server image
```bash
docker pull seedplatform/bsyncr-server:latest
```

Run the container by following steps 1 and 2 in the "how to test manually" section here:
https://github.com/SEED-platform/seed/pull/2498

Note that you don't have to set the `CELERY_*` variables mentioned in step 2.

To test, create and run a bsyncr analysis using different model types: SLR, 3 param cooling, 3 param heating, and 4 param using the New Analysis modal for a property that's been properly setup for bsyncr: it must have latitude, longitude, and 12+ electricity meter readings.

All analyses should run successfully, and should show the derived model JSON when viewing the results detail page. In addition, you should see both a PNG and BuildingSync file in the Output Files column of the results table. Clicking on the PNG should show you the plot generated by bsyncr

#### What are the relevant tickets?

#### Screenshots (if appropriate)
<img width="1428" alt="Screen Shot 2021-01-12 at 7 34 58 PM" src="https://user-images.githubusercontent.com/18518728/104391160-4ece2280-550d-11eb-9c9f-7e64d9441c77.png">
